app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/Constants.java

```diff
@@ -4,7 +4,7 @@
  * Various constants
  */
 public class Constants {
-    public static final double MMOLL_TO_MGDL = 18.0182;
+    public static final double MMOLL_TO_MGDL = 18.0156;
```

Comment:
Please insert the following comment before the declaration:
```Java
// Constant is based on the official molecular weight (see https://pubchem.ncbi.nlm.nih.gov/compound/glucose).
```

Reply:
Done. I've tweaked the comment a little bit because NCBI is no more official than several other sources (and also they specify one digit less). I've also made a note about the other (wrong as far as I can tell) constant being used in other places, which may clear up confusion in future or be useful when someone who is writing a new app / text considers where to copy their constant value from.

WDYT?